PowerEditor/installer/nativeLang/german.xml

```diff
@@ -947,7 +957,7 @@ Mehrere Spaltenmarkierungen können definiert werden, indem die verschiedenen Za
 				</Language>
 
 				<Highlighting title="Hervorhebung">
-					<Item id="6351" name="Alle markieren"/>
+					<Item id="6351" name="Alle Vorkommen der Markierung stilisieren"/>
```

Comment:
Die Übersetzung mit der ID 6351 ist der Titel einer Gruppierungsbox in der Rubrik "Hervorhebung" des Einstellungsdialogs. Warum also hier die Terminologie wechseln? Besser (da konsistent) wäre es doch, hier bei "Alle Hervorheben" zu bleiben.